runtime/integration-tests/src/runtime_apis/pools.rs

```diff
@@ -0,0 +1,187 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use cfg_primitives::{AccountId, CollectionId, PoolId, CFG};
+use cfg_traits::{Permissions, PoolMutate};
+use cfg_types::{
+	fixed_point::Rate,
+	permissions::{PermissionScope, PoolRole, Role},
+	tokens::{CurrencyId, CustomMetadata},
+};
+use development_runtime::{apis::PoolsApi, RuntimeOrigin};
+use frame_benchmarking::account;
+use frame_support::{
+	assert_ok,
+	dispatch::RawOrigin,
+	traits::{
+		tokens::nonfungibles::{Create, Mutate as NMutate},
+		OriginTrait,
+	},
+};
+use orml_traits::asset_registry::Mutate;
+use pallet_loans::types::Asset;
+use pallet_pool_system::tranches::{TrancheInput, TrancheMetadata, TrancheType};
+use sp_core::{bounded::BoundedVec, sr25519, Pair};
+use sp_runtime::{
+	traits::{IdentifyAccount, One},
+	Perquintill,
+};
+use tokio::runtime::Handle;
+
+use super::ApiEnv;
+use crate::pools::utils::loans::LoanId;
+
+#[tokio::test]
+async fn test() {
+	ApiEnv::new(Handle::current())
+		.startup(|| {
+			let pool_id = 3;
+
+			let admin = sp_runtime::AccountId32::from(
+				<sr25519::Pair as sp_core::Pair>::from_string("//Alice", None)
+					.unwrap()
+					.public()
+					.into_account(),
+			);
+
+			let borrower = sp_runtime::AccountId32::from(
+				<sr25519::Pair as sp_core::Pair>::from_string("//Bob", None)
+					.unwrap()
+					.public()
+					.into_account(),
+			);
+
+			let admin_account = account::<AccountId>("//Alice", 0, 0);
+			let borrower_account = account::<AccountId>("//Bob", 0, 0);
+
+			<development_runtime::Balances>::set_balance(
```

Comment:
please instead of using the development runtime explicitly user the import `crate::chain::centrifuge::PALLET_NAME_HERE`. This will automatically import the right runtime.

Reply:
I used the `rewards` runtime api test as a sample and thought there was a reason we are using explicitly `development_runtime` there, since it was approved just a few weeks ago! Will change!